build.sh

```diff
@@ -102,25 +102,34 @@ check_sdk_version () {
     major="${semver[0]:-0}"
     minor="${semver[1]:-0}"
     patch="${semver[2]:-0}"
+    release="${semver[3]:-0}"
+
+    sslverify=true
 
     if [ ! -z "${SDK_VER}" ]; then
         sdk_ver=$SDK_VER
         return 0
     fi
 
     if [ $major -ge 4 ]; then
-        if [ $minor -ge 9 ]; then
-            # for v4.9.x.x use newer SDK
-            sdk_ver=1.52.1
+        if [ $minor -ge 14 ] && ( [ $patch -ge 1 ] || [ $release -ge 5 ] ); then
```

Comment:
patch version is always zero for mailine releases also if we pass version 4.15.0.1 it will not be built with the new toolchain because patch > 1 and release > 5 will fail.